### ISSUE TITLE
This tool is amazlingly well done. Are there any plans for a macos release?

### ISSUE THREAD

**Issue by tracure1337** [marcopaganini/clipsync#9]
This tool is amazlingly well done. Are there any plans for a macos release?

That'd be so cool !

Thanks for putting time and effort into this at all!

**Comment by marcopaganini** [comment]
Hi There,

I'd like to do that but there's a problem: I don't have access to a MacOS machine :)

There are some differences, as far as I know, but since I don't have access, I can't confirm. IIUC, it appears that Macs (like Windows) don't have a "primary" area, just the clipboard. This in a way makes it simpler, but would need some modifications to the code. I also think I saw a "universal" clipboard handling library in Go and I'm assuming here it would work on Macs (on Linux, it uses xclip, I don't recall how it does it on macs).

So, TL;DR: If I had access (remote or local) to a MacOS device, I could try to do it.
